lab-portal/services/auditService.tsx

```diff
@@ -1,6 +1,6 @@
 import axios from 'axios';
 
-const API_BASE_URL = 'https://localhost:7282/api';
+const API_BASE_URL = `${process.env.EXPO_PUBLIC_API_URL}:7282/api`;
```

Comment:
Because the api can have dynamic ports, this should not be committed to the repo. It exposes which port the api is using which should not be public. I would also put https://localhost:7282/api part of the .env so that you don't have to put /api everywhere. It's a waste of space.